StardewArchipelago/GameModifications/CodeInjections/Bundles/BundleMenuInjection.cs

```diff
@@ -0,0 +1,194 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StardewModdingAPI;
+using StardewValley;
+using StardewValley.Extensions;
+using StardewValley.ItemTypeDefinitions;
+using StardewValley.Menus;
+
+namespace StardewArchipelago.GameModifications.CodeInjections.Bundles
+{
+    public class BundleMenuInjection
+    {
+        private static IMonitor _monitor;
+        public static void setupLogging(IMonitor monitor)
+        {
+            _monitor = monitor;
+        }
+        
+        //This is what a replacement method would be
+        public static bool SetUpBundleSpecificPage(JunimoNoteMenu __instance, Bundle b)
```

Comment:
I see your point, and I agree the transpiler is much, much cleaner.

I have been toying with the idea of making my custom JunimoNoteMenu for a while. If I did, I could include your changes in it and get rid of the transpiler. But I don't plan on doing that now, so I don't believe it's relevant to you. Just informing you that this is one more drop in that bucket that will eventually tip over one evening that I'm bored.